### DISCUSSION TITLE
Post new discussions automatically to #developers on Ark's discord

### DISCUSSION THREAD

**Discussion by c0nsol3** [ArkEcosystem/core#4530] [Ideas]
In order to remind people about Ark's Discussion on GH it may be worthwhile to mage a post in #developers on Ark's discord every time a new new discussion is started or an answer is selected.

You think this would be useful/worth it or a waste of resources?

**Comment by c0nsol3** [comment]
Besides the awareness of GH Discussions this could also remind people within the community to comment on discussion for which answer that may know or issues they are experiencing or have experienced in the past.

**Comment by faustbrian** [comment]
> You think this would be useful/worth it or a waste of resources?

If there is an existing solution that can do this to which you can link we can look at installing it but otherwise we won't be allocating any resources to this at the moment.
